Tests/js/views/tabs/assets/ez-locationviewdetailstabview-tests.js

```diff
@@ -201,6 +254,145 @@ YUI.add('ez-locationviewdetailstabview-tests', function (Y) {
+                        return {Owner: that.owner};
+                    } else {
+                        Y.fail("Unexpected parameter " + attr + " for content mock");
+                    }
+                }
+            });
+
+            Mock.expect(this.contentMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            Mock.expect(that.versionMock, {
+                method: 'get',
+                args: ['resources'],
+                returns: {Creator: this.creator}
+            });
+
+            Mock.expect(this.versionMock, {
+                method: 'getTranslationsList',
+                returns: this.translationsList
+            });
+
+            Mock.expect(this.versionMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+            Mock.expect(this.locationMock, {
+                method: 'toJSON',
+                returns: {}
+            });
+
+
+            this.view = new Y.eZ.LocationViewDetailsTabView({
+                content: this.contentMock,
+                location: this.locationMock,
+                selectedSortField: '',
+                selectedSortOrder: '',
+                container: '.container'
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+
+        "Should set the selectedSortField and selectedSortOrder attributes": function () {
+            Assert.areSame(
+                this.sortField,
+                this.view.get('selectedSortField'),
+                "selectedSortField should be set"
+            );
+
+            Assert.areSame(
+                this.sortOrder,
+                this.view.get('selectedSortOrder'),
+                "selectedSortOrder should be set"
+            );
+        },
+
+        "Should change selectedSortField and selectedSortOrder when changing the select tag": function () {
+            this.view.render();
+            var container = this.view.get('container'),
+                sortFieldSelector = container.one('.ez-subitems-ordering-sort-type'),
+                selectedSortFieldIndex = sortFieldSelector.get('selectedIndex'),
+                sortOrderSelector = container.one('.ez-subitems-sorting-order'),
+                selectedSortOrderIndex = sortOrderSelector.get('selectedIndex'),
+                newSortField = 'SECTION',
+                newSortOrder = 'DESC';
+
+            Assert.areNotSame(
+                newSortField,
+                this.view.get('selectedSortField'),
+                "selectedSortField should not be changed yet"
+            );
+
+            Assert.areNotSame(
+                newSortOrder,
+                this.view.get('selectedSortOrder'),
+                "selectedSortOrder should not be changed yet"
+            );
+
+            sortFieldSelector.get('options').item(selectedSortFieldIndex).set('value', newSortField);
```

Comment:
related to https://github.com/ezsystems/PlatformUIBundle/pull/517/files#r54079441, a new options should be selected, you should not just change the value of the selected one.